week1/414_thrid_maximum.py

```diff
@@ -0,0 +1,16 @@
+# https://leetcode.com/problems/third-maximum-number/
+def third_maximum(nums: list[int]) -> int:
+    max_num = 3
```

Comment:
This might be controversial, but I think it's okay not to make this variable for readability. using 3 as a literal in the 5th line seems fine to me.